### PR TITLE
Adding the creation of public_network

### PR DESCRIPTION
**Pull request by cedlecomte**

### CODE REVIEW

**Comment by cedlecomte** [comment]
Maybe we can put the range for the public_network in parameters. Tell me if it make sense, I will add a new parameter.

File: virtualizor.py [hunk 1]
```diff
@@ -105,17 +105,13 @@ def __init__(self, conf):
...
-            logging.error("public_network %s should be active and have "
-                          "internal DHCP turned on." % (conf.public_network))
-            raise self.MissingPublicNetwork()
-
     def create_networks(self, conf, install_server_info):
         net_definitions = {
-            ("%s_sps" % conf.prefix): {}
+            ("%s_sps" % conf.prefix): {},
+            ("%s" % conf.public_network): 
+                {"dhcp":{"address":"10.0.0.1","netmask":"255.255.255.0",
```
Comment (goneri) [inline virtualizor.py]: I think a 192.168.140.0/24 is more likely to be unused. It's a bit common to see the full 10/8 used.

File: virtualizor.py [hunk 2]
```diff
@@ -425,6 +424,9 @@ class Network(object):
       <range start='{{ host.ip }}' end='{{ host.ip }}' />
```
Comment (goneri) [inline virtualizor.py]: The line above is not used anymore.

**Comment by goneri** [comment]
@Prophidys , please take a look on the unit-test too.

**Comment by goneri** [comment]
:+1: for me.

**Pull request merged**